src/modules/bug-reports.ts

```diff
@@ -28,25 +28,33 @@ export const createReport = (
         description,
         townhallId,
         submitterId: user._id,
+        resolved: false,
+        replies: [],
     });
 };
 
 /**
  * @description Retrieves at most 10 reports from the bug-reports collection depending on the page number.
  * @param {number} page - Page number to return. If the page number exceeds the number of available pages, 0 reports are returned.
- * @param {string} ascending - Describes the sorted order of the reports.'True' for ascending. 'False' for descending.
+ * @param {boolean} sortByDate - Sort by date order. True for ascending. False for descending.
+ * @param {boolean} resolved - Resolved status of reports to retrieve
  * @returns {Promise<BugReport[]>} - promise that will produce an array of bug reports.
- * @
  */
 const numberOfDocumentsPerPage = 10;
 export const getReports = (
     page: number,
-    ascending: string
+    sortByDate: boolean,
+    resolved?: boolean
 ): Promise<BugReport[]> => {
+    const resolvedQuery =
+        typeof resolved === 'boolean'
+            ? { resolved }
+            : { $or: [{ resolved: true }, { resolved: false }] };
```

Comment:
I think this could be optimized.  I'm not sure if there's a purpose to the else part of the ternary condition.
```ts
const query = {};
if (typeof resolved === 'boolean') query.resolved = resolved; 
// else we just leave the query as an empty {} because that defaults to finding all of them
```

Reply:
thanks for letting me know about the optimization